castanet/counter.py

```diff
@@ -43,7 +50,14 @@ def count_whileloops(cast_dict):
 
 
 def count_forloops(cast_dict):
-    """A function for counting the number of while loops in a Python program."""
+    """A function for counting the number of for loops in a Python program.
+
+    Args:
+        cast_dict: adds the number of for loops to a dictionary
```

Comment:
This should read `cast_dict: A dictionary of files and the corresponding CAST's`.